Libraries/Image/resolveAssetSource.js

```diff
@@ -104,9 +110,13 @@ function getBasePath(asset) {
  */
 function getScaledAssetPath(asset) {
```

Comment:
I see. So supposing the app is sideloaded at `file://apps/foobar/sideload/main.jsbundle` , an asset would look like `file://apps/foobar/sideload/assets/drawable_mdpi/uikit_button_special_icon`

On iOS, we actually use the JS folder structure, so the asset would be at `file://apps/foobar/sideload/assets/drawable_mdpi/uikit/button/special/icon.png`

I suppose to keep platform conventions we should keep these as being different. What do you think @frantic / @nicklockwood ?

At any rate, can you mention these examples in the PR description, and put an example inline in a comment in `getScaledAssetPath`. This will vastly help folks understand what's going on here in the future.